dotCMS/src/main/java/com/dotcms/rest/api/v1/authentication/ApiTokenResource.java

```diff
@@ -260,11 +509,95 @@ private User getUserById(ApiTokenForm formData, User requestingUser) {
+                                            "      \"expiresDate\": 0 \n," +
+                                            "      \"id\": \"string\",\n" +
+                                            "      \"issueDate\": 0,\n" +
+                                            "      \"issuer\": \"string\",\n" +
+                                            "      \"modificationDate\": 0,\n" +
+                                            "      \"notBeforeDate\": false,\n" +
+                                            "      \"requestingIp\": \"string\",\n" +
+                                            "      \"requestingUserId\": \"string\",\n" +
+                                            "      \"revoked\": false,\n" +
+                                            "      \"revokedDate\": 0,\n" +
+                                            "      \"subject\": \"string\",\n" +
+                                            "      \"tokenType\": \"string\",\n" +
+                                            "      \"userId\": \"string\",\n" +
+                                            "      \"valid\": true\n" +
+                                            "    }\n" +
+                                            "  },\n" +
+                                            "  \"errors\": [],\n" +
+                                            "  \"i18nMessagesMap\": {},\n" +
+                                            "  \"messages\": [],\n" +
+                                            "  \"permissions\": []\n" +
+                                            "}"
+                                        )
+                                    }
+                                )
+                    ),
+                    @ApiResponse(responseCode = "400", description = "Bad request"),
+                    @ApiResponse(responseCode = "403", description = "Forbidden"),
+                    @ApiResponse(responseCode = "500", description = "Unexpected server error")
+                }
+    )
+    public final Response getRemoteToken(
+            @Context final HttpServletRequest httpRequest,
+            @RequestBody(description = "PUT body consists of a JSON object containing the following properties:\n\n" + 
+                                        "| **Property** | **Value** | **Description**                      |\n" +
+                                        "|--------------|-----------|--------------------------------------|\n" +
+                                        "| `userId`     | String    | ID of user attempting receiving      |\n" +
+                                        "| `tokenId`    | String    | ID of api token user is getting      |\n" +
+                                        "| `label`      | String    | Name of requested token              |\n" +
+                                        "| `revoked`    | Boolean   | Value for if token is revoked or not |",
```

Comment:
This might need a revisit; looks like the properties specified here don't match the ones given in the example immediately below.